### PR TITLE
♿️ [PRTL-1576] suggestion facet 508

### PR DESCRIPTION
**Pull request by Jephuff**

### CODE REVIEW

File: src/packages/@ncigdc/components/Aggregations/SuggestionFacet.js [hunk 1]
```diff
@@ -199,10 +199,14 @@ const SuggestionFacet = compose(
                       onKeyDown={selectableList.handleKeyEvent}
                       placeholder={placeholder}
                       value={inputValue}
-                      aria-activedescendant={_.get(
-                        selectableList,
-                        'focusedItem.id',
-                      )}
+                      aria-activedescendant={
+                        active
```
Comment (alex-wilmer) [inline src/packages/@ncigdc/components/Aggregations/SuggestionFacet.js]: does `undefined` make a difference here vs false / null ?

Comment (Jephuff) [inline src/packages/@ncigdc/components/Aggregations/SuggestionFacet.js]: `null` works 👍 updated

Comment (alex-wilmer) [inline src/packages/@ncigdc/components/Aggregations/SuggestionFacet.js]: Oh i meant why not `active && ...`

Comment (Jephuff) [inline src/packages/@ncigdc/components/Aggregations/SuggestionFacet.js]: Unfortunately false gets turned into a string so it won't work. Has to be either null or undefined.

Comment (alex-wilmer) [inline src/packages/@ncigdc/components/Aggregations/SuggestionFacet.js]: gotcha. can you leave a comment there about that?

Comment (alex-wilmer) [inline src/packages/@ncigdc/components/Aggregations/SuggestionFacet.js]: someone like me might come and erase that later not knowing

Comment (Jephuff) [inline src/packages/@ncigdc/components/Aggregations/SuggestionFacet.js]: definitely! :) 
done.

**Pull request merged**